README.rst

```diff
@@ -90,8 +90,69 @@ testing with live email addresses.  It works like this:
     client.destroy_account(acct.email, "MySecretPassword")
 
 
+Passing tokens and assertions to other applications
+===================================================
+
+PyFxA provides a ``fxa-client`` that you can use to export Bearer
+Tokens and Browser ID assertions.
+
+
+Get a Bearer Token for an existing account
+------------------------------------------
+
+.. code-block:: bash
+
+    fxa-client --bearer --auth you@domain.tld \
```

Comment:
The problem with subcommand is that it is not trivial to have arguments that works for all subcommands and some that works with some subcommands.

The good think with ``--browserid`` and ``--bearer`` arguments is that you can actually generate both at the same time:

    fxa-client --create --bearer --browserid